test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/StructuralTypeConfigurationTest.cs

```diff
@@ -188,5 +188,39 @@ public void NamespaceAssignment_AutoAssignsNamespaceToStructuralType_NamespaceAs
             Assert.Equal(expectedNamespace, modelBuilder.EntityType<OrderHeader>().Namespace);
             Assert.Equal(expectedNamespace, modelBuilder.ComplexType<ZipCode>().Namespace);
         }
+
+        /// <summary>
+        /// Tests the Namespace property setter logic with a null or white space value throws an argument exception.
```

Comment:
@icnocop Instead of having 3 asserts for 3 different scenarios, we could use:
```csharp
[Theory]
[InlineData(null)]
[InlineData("")
[InlineData(" ")]
public void NamespaceAssignment_WithNullOrWhiteSpace_ThrowsArgumentException(string namespace)
{
    // ...
    ExceptionAssert.ThrowsArgument(() => entityTypeConfiguration.Namespace = namespace, "value");
}
```

Reply:
Fixed. Thank you.